README.md

```diff
@@ -36,8 +35,8 @@ on:
 
 env:
   FLY_API_TOKEN: ${{ secrets.FLY_API_TOKEN }}
-  FLY_REGION: iad
-  FLY_ORG: personal
+  FLY_REGION: <put your desired region>
+  FLY_ORG: <put your fly org name>
```

Comment:
I think it's better to show an example instead of explaining what to put. Please revert this two lines.